### PR TITLE
HDFS scheduler uris as a JSON array.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
Attention @adam-mesos @elingg

### CODE REVIEW

File: repo/packages/H/hdfs/0/config.json [hunk 1]
```diff
@@ -13,10 +13,15 @@
       "description": "The zookeeper URL(s) to fill into hdfs-site.xml for the NameNodes and DataNodes.",
       "type": "string"
     },
-    "hdfs/scheduler-uri": {
+    "hdfs/scheduler-uris": {
```
Comment (jsancio) [inline repo/packages/H/hdfs/0/config.json]: Should this be required? We schema check the "merged" values, no?

**Comment by jsancio** [comment]
Looks good. Just a minor comment.

File: repo/packages/H/hdfs/0/config.json (hunk 1 above)
Comment (ConnorDoyle) [inline repo/packages/H/hdfs/0/config.json]: Yes, but we also merge in the defaults, so it shouldn't be marked required.

**Comment by ConnorDoyle** [comment]
:ship:ing it

File: repo/packages/H/hdfs/0/config.json (hunk 1 above)
Comment (jsancio) [inline repo/packages/H/hdfs/0/config.json]: Ship it.

File: repo/packages/H/hdfs/0/config.json [hunk 2]
```diff
@@ -13,10 +13,15 @@
...
       "description": "The URI of the hdfs-mesos tarball, which will be extracted and used to run the scheduler.",
-      "type": "string",
-      "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/ovh/hdfs-mesos-0.1.0.tgz"
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": [
+        "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/ovh/hdfs-mesos-0.1.0.tgz"
```
Comment (adam-mesos) [inline repo/packages/H/hdfs/0/config.json]: Still want to update this to the dcos build (and add disk resources), but I'll do that in a follow-up PR

Comment (ConnorDoyle) [inline repo/packages/H/hdfs/0/config.json]: SGTM

**Pull request merged**